### ISSUE TITLE
Create Advanced Tutorial - lv_buses

### ISSUE THREAD

**Issue by samuraisoldier** [e2nIEE/pandapower#354]
When trying to recreate the network in the advanced network tutorial i think lv_buses is not declared. 

In cell 57, lv_buses is called but was not saved in cell 56 so this produces an error.

I believe this:
```
# show only low voltage buses
net.bus[net.bus.vn_kv == 0.4]
```

should be:
```
# show only low voltage buses
lv_buses = net.bus[net.bus.vn_kv == 0.4]
lv_buses
```

Thanks

**Comment by lthurner** [comment]
Yes you are right about that. Do you want to put in a PR to fix this?

**Comment by lthurner** [comment]
Fixed with #379
